src/actions/appActions.js

```diff
@@ -268,6 +269,10 @@ export const setupSentryAction = (user: ?Object, wallet: Object) => {
         ethAddress: address,
       },
     });
+    // eslint-disable-next-line i18next/no-literal-string
+    Instabug.setUserAttribute('address', wallet.address);
+    if (wallet.provider._network.ensAddress) {
+      Instabug.setUserAttribute('ENS', wallet.provider._network.ensAddress);
+    }
```

Comment:
```suggestion
    if (username) {
      Instabug.setUserAttribute('ENS', username);
    }
```

Suggesting as we have already destructured `username` from the incoming `user` parameter.

Reply:
Sure will do that.